startup.sql

```diff
@@ -0,0 +1,41 @@
+/* This script creates the database for capurace. */
+/* The description of the database is in the wiki, Database-Structure.md */
+
+CREATE DATABASE IF NOT EXISTS capurace;
+USE capurace;
+
+CREATE TABLE IF NOT EXISTS user (
+        school VARCHAR(30) NOT NULL DEFAULT NULL UNIQUE KEY,
+        id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        leader VARCHAR(10) NOT NULL DEFAULT NULL,
+        tel VARCHAR(11) NOT NULL DEFAULT NULL,
+        mail VARCHAR(30) NOT NULL DEFAULT NULL,
+        password VARCHAR(30) NOT NULL DEFAULT NULL,
+        nteams SMALLINT UNSIGNED NOT NULL DEFAULT 0,
+        bill INT NOT NULL DEFAULT 0,
+        paid SMALLINT NOT NULL DEFAULT 0
```

Comment:
MySQL seems to have `Boolean` type [1] , though it's just synonyms for TINYINT. Maybe you can use TINYINT here? Just for jiepi. ;-)

[1] http://dev.mysql.com/doc/refman/5.0/en/numeric-type-overview.html